public/video-ui/src/components/FormFields/DatePicker.js

```diff
@@ -3,11 +3,12 @@ import Picker from 'react-datepicker';
 import moment from 'moment';
 import 'react-datepicker/dist/react-datepicker.css';
 import Icon from '../Icon';
+import _range from 'lodash/fp/range';
 
-const DATE_FORMAT = 'YYYY/MM/DD';
+const DATE_FORMAT = 'DD MMM YYYY';
 const DATETIME_FORMAT = `${DATE_FORMAT} HH:mm`;
 
-const MINUTES = [0, 15, 30, 45].map(minute =>
+const MINUTES = _range(0, 60).map(minute =>
```

Comment:
We probably also want to change the expiry lambda to run every minute now too, as opposed to [every 15 minutes](https://github.com/guardian/media-atom-maker/blob/master/cloudformation/media-atom-maker.yaml#L521).